Contentstack.Core.Tests/EntryTest.cs

```diff
@@ -98,6 +98,8 @@ await sourceEntry
                     else
                     {
                         Assert.True(result.Uid == sourceEntry.Uid);
+                        Assert.NotNull(result._variant);
+                        Assert.NotNull(result._variant["uid"]);
```

Comment:
@nadeem-cs Will result._variant["uid"] return a response? 
new response format is below. Use result._variant["_uid"]
```
"_variant": {
       "_uid": "variant uid",  
       "_instance_uid": "entry_variant uid",    
       "_change_set": [],
       "_base_entry_version": 'version number of base entry',
    }
```